test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/LoadTest.java

```diff
@@ -46,7 +47,7 @@
 	public static OptionalInt hcsSubmitMessage = OptionalInt.empty();
 	public static OptionalInt hcsSubmitMessageSizeVar = OptionalInt.empty();
 	/** initial balance of account used as sender for performance test transactions */
-	public static OptionalLong initialBalance = OptionalLong.of(ONE_MILLION_HBARS);
+	public static OptionalLong initialBalance = OptionalLong.of(DEFAULT_INITIAL_BALANCE);
```

Comment:
You added a new constant, does the CI property initialBalance not work?

Reply:
`DEFAULT_INITIAL_BALANCE` will be more clear and aligns with other DEFAULT static variables in the class. This value can be used in any other class. `DEFAULT_INITIAL_BALANCE` is assigned to `ONE_MILLION_HBARS` in `PerfTestLoadSettings`

Reply:
OK, make sense.